packages/@aws-cdk-testing/framework-integ/test/aws-ecs-patterns/test/fargate/integ.alb-fargate-service-command-entry-point.ts

```diff
@@ -13,22 +13,28 @@ const stack = new cdk.Stack(
 // Create VPC and cluster
 const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 2, restrictDefaultSecurityGroup: false });
 const cluster = new ecs.Cluster(stack, 'TestFargateCluster', { vpc });
+const securityGroup = new ec2.SecurityGroup(stack, 'SecurityGroup', {
+  vpc,
+  allowAllOutbound: true,
+});
+securityGroup.addIngressRule(ec2.Peer.anyIpv4(), ec2.Port.tcpRange(32768, 65535));
```

Comment:
Nit: Could we restrict the inbound rules to allow only specific container port and also to deny all outbound traffic (`allowAllOutbound: false`) as I don't think it will be required for this usecase.